app/components/container/containerController.js

```diff
@@ -116,9 +116,32 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
     });
   };
 
-  $scope.remove = function () {
+  $scope.confirmRemove = function () {
+    if ($scope.container.State.Running) {
+      ModalService.confirm({
```

Comment:
You can actually use the `ModalService.confirmDeletion` method instead of `ModalService.confirm` here.